src/components/FileTreeNode/index.spec.tsx

```diff
@@ -178,4 +186,34 @@ describe(__filename, () => {
 
     expect(renderChildNodes).toHaveBeenCalled();
   });
+
+  it('marks a file node as selected', () => {
+    const version = createInternalVersion(fakeVersion);
+
+    const renderProps = getTreefoldRenderProps({
+      id: version.selectedPath,
+    });
+
+    const root = render({
+      ...renderProps,
+      version,
+    });
+
+    expect(root.find(`.${styles.selected}`)).toHaveLength(1);
+  });
+
+  it('does not mark a file node as selected when it is not selected', () => {
+    const version = createInternalVersion(fakeVersion);
+
+    const renderProps = getTreefoldRenderProps({
+      id: `src/${version.selectedPath}`,
```

Comment:
This is a little confusing. The test is supposed to set `id` to an unrelated path, right? I think it would be less confusing if it had no relation at all.

Reply:
Otherwise, you could add a comment saying that it shouldn't be selected since the parent directory differs.

Reply:
I changed to an `id` that is totally unrelated.

Reply:
I changed to an id that is totally unrelated.